.github/workflows/test.yaml

```diff
@@ -35,6 +38,13 @@ jobs:
       matrix:
         os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
         python-version: [3.7, 3.8, 3.9]
+        include:
+        - os: ubuntu-latest
+          path: /usr/share/miniconda3/envs/
+        - os: macos-latest
+          path: /Users/runner/miniconda3/envs/
+        - os: windows-latest
+          path: C:\Miniconda3\envs\
```

Comment:
I've seen that these absolute paths have been removed from the [documentation of the setup-miniconda action](https://github.com/conda-incubator/setup-miniconda). Maybe they're no longer required?